src/main/java/com/google/sps/data/ProcessCareerQuizResults.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.HashMap; 
+import java.util.Map; 
+import java.util.Map.Entry;
+import com.google.sps.data.CareerQuestionChoice;
+
+/** Returns the recommended career path based off the career quiz results. */
+public class ProcessCareerQuizResults {
+  private ProcessCareerQuizResults() {};
+  
+  public static String getRecommendedCareerPath(List<CareerQuestionChoice> userChoices) {
```

Comment:
This looks a lot like my interview question :)

Reply:
;)